<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OPOLaserControlControls</class>
 <widget class="QWidget" name="OPOLaserControlControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>OPOTEK Phocus Control</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>301</width>
          <height>431</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="checkBoxTriggerExternally">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Trigger Externally</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonInitLaser">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Init Laser</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="buttonFlashlamp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Start Lamp</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonQSwitch">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Start Laser</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelStatus">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Laser Status</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Current Wavelength</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelWavelength">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
               <stylestrategy>PreferDefault</stylestrategy>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>750.0 nm</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Pulse Energy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelEnergy">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
               <stylestrategy>PreferDefault</stylestrategy>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>0 mJ</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="sliderWavelength">
           <property name="minimum">
            <number>6900</number>
           </property>
           <property name="maximum">
            <number>9500</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="pageStep">
            <number>100</number>
           </property>
           <property name="sliderPosition">
            <number>7500</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDoubleSpinBox" name="spinBoxWavelength">
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="minimum">
              <double>690.000000000000000</double>
             </property>
             <property name="maximum">
              <double>950.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
             <property name="value">
              <double>750.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>nm</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonTune">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Tune</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Fast Tuning Wavelengths</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableFastTuningWavelengths">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::SolidLine</enum>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>42</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>50</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <row>
            <property name="text">
             <string>Î»/nm</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>2</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>3</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>4</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>5</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>6</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>7</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>8</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>9</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>10</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>11</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>12</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>13</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>14</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>15</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>16</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>17</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>18</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>19</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>20</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>21</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>22</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>23</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>24</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>25</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>26</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>27</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>28</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>29</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>30</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>31</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>32</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>33</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>34</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>35</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>36</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>37</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>38</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>39</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>40</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>690</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>690</string>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string>690</string>
            </property>
           </item>
           <item row="0" column="3">
            <property name="text">
             <string>690</string>
            </property>
           </item>
           <item row="0" column="4">
            <property name="text">
             <string>690</string>
            </property>
           </item>
           <item row="0" column="5">
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item row="0" column="6">
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item row="0" column="7">
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item row="0" column="8">
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item row="0" column="9">
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item row="0" column="10">
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item row="0" column="11">
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item row="0" column="12">
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item row="0" column="13">
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item row="0" column="14">
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item row="0" column="15">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="16">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="17">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="18">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="19">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="20">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="21">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="22">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="23">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="24">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="25">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="26">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="27">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="28">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="29">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="30">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="31">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="32">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="33">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="34">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="35">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="36">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="37">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="38">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="0" column="39">
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonFastTuning">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Start Fast Tuning</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="checkBoxCalibration">
             <property name="text">
              <string>calibrate OPO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxPosition">
             <property name="maximum">
              <number>50000</number>
             </property>
             <property name="singleStep">
              <number>400</number>
             </property>
             <property name="value">
              <number>24400</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
